router/signup.js

```diff
@@ -1,12 +1,10 @@
+const renderTemplate = require("../util/renderTemplate");
 const express = require("express");
 const router = express.Router();
 
 
 router.get("/signup", function(req, res) {
-	res.render("template", {
-		page: "signup",
-		pageArg: null,
-	});
+	renderTemplate(res, "Login", "login");
```

Comment:
It looks like the "/signup" route is rendering the "login" page here, it should probably render "signup".